Index: Client.cpp
===================================================================
--- Client.cpp  (revision 59)
+++ Client.cpp  (working copy)
@@ -127,7 +127,7 @@
        }

        unsigned int mode = *((unsigned int*)data);
-       if(mode == (unsigned int)LoginMode::lm_from_world)
+       if(mode == (unsigned int) lm_from_world)
        {
                log->Log(log_network, "Session ready indicated logged in from world(unsupported feature), disconnecting.");
                connection->Close();
